index.md

```diff
@@ -57,7 +57,7 @@ If you have questions, please see the [FAQ][]. If that doesn't answer your quest
 
 ## Reporting Guidelines
 
-If you believe someone is violating the Code of Conduct, we ask that you report it by emailing [tc39-conduct-reports@googlegroups.com][tc39-conduct]. All reports will be kept confidential. Only members of the [Code of Conduct Committee][], and possibly the Ecma ExeCom, will receive the reports. No identity will be made public before the individuals concerned have agreed to it.
+If you believe someone is violating the Code of Conduct, we ask that you report it by emailing [tc39-conduct-reports@googlegroups.com][tc39-conduct]. **All reports will be kept confidential.**
```

Comment:
Should probably link to the confidentiality section below.

Reply:
Done